conftest.py

```diff
@@ -38,3 +50,13 @@ def env(request):
 @pytest.fixture(scope="module")
 def apiversion(request):
     return request.config.getoption("--api-version")
+
+
+@pytest.fixture(scope="module")
+def qacollectionuser(request):
+    return request.config.getoption("--qa_collection_user")
```

Comment:
No need to use `request` as there's a [pytest fixture for config](https://docs.pytest.org/en/latest/reference.html#pytestconfig):

```python
def qacollectionuser(pytestconfig):
    return pytestconfig.getoption('qa_collection_user')
```

Also, I'd make this session scoped as it doesn't change per module.